### ISSUE TITLE
Fluid registry issue with Mariculture and TSteelworks: Value Already Present.

### ISSUE THREAD

**Issue by belathus** [Ephys/TinkersSteelworks#106]
http://pastebin.com/RU6GBDia

It was not a crash. And interestingly, with both mods installed, I can easily start the server or start a single player world. I cannot, however, join a server. Removing TSteelworks does fix this issue, but I'd rather not if I can help it.

**Comment by Ephys** [comment]
This might be mariculture's fault, because I only register liquid limestone if it doesn't already exist https://github.com/Ephys/TinkersSteelworks/blob/master/src/main/java/toops/tsteelworks/lib/ModsData.java#L90

**Comment by belathus** [comment]
I'm not super familiar with modding, but it looks like Mariculture registers the fluid as 'quicklime'. Line 200 here: https://github.com/joshiejack/Mariculture/blob/d295f75b282471c15cef958f0aa7bf0f0973e25a/src/main/java/mariculture/core/Core.java

He also uses a fluid helper that also appears to check if the fluid is already registered here: https://github.com/joshiejack/Mariculture/blob/7f69088640dbf2d282e1664bc9e858777be8f281/src/main/java/mariculture/core/helpers/FluidHelper.java

In my modpack, I know there is a number of fluids defined as custom items, but it does not look like there is one for molten limestone.

**Comment by belathus** [comment]
Yeah, I have no idea. I removed Mariculture; now I'm getting the same error about moltenaeonsteel, from Tinkers' Defense. I know TSteelworks doesn't register that, but everything works when I remove TSteelworks. And, of course, it still works in single player.
